integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java

```diff
@@ -12,9 +12,9 @@
 import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import io.kubernetes.client.openapi.ApiException;
 import io.kubernetes.client.openapi.Configuration;
-import io.kubernetes.client.openapi.apis.ApiextensionsV1Api;
+import io.kubernetes.client.openapi.apis.ApiextensionsV1beta1Api;
```

Comment:
You are unintentionally reverting a change here to move from the "v1beta1" API's to the modern "v1" versions.

Reply:
Yes I remember this change :-) Thanks for catching. Fixed